scania-digital-design-system/tegel
# fix(dropdown): text overflow in dropdown-option

**Describe pull-request**  
This PR introduces a `word-wrap:anywhere` to allow for the CSS to wrap any dropdown-option content when there is an overflow. 

**Solving issue**  
Fixes: [CDEP-2638](https://tegel.atlassian.net/browse/CDEP-2638)

**How to test**  
1. Go to Dropdown
2. Add a long word in an option: "LÅÅÅÅÅÅÅÅÅÅÅÅÅÅÅÅÅÅÅÅÅÅNGTOOOOOOOORD"
3. Make sure it breaks correctly.
